WEB_SERVICE/WEB/_MEDIUM/WEB_XSS/description.md

```diff
@@ -13,3 +13,65 @@ XSS is commonly separated into three families
 
 XSS vulnerabilities allow an attacker to perform a variety of malicious actions, like exfiltration of personal data,
 including user session or account information; perform actions on behalf of the user.
+
+
+### Examples
```

Comment:
These are not related to this PR?

Reply:
all the PRS were merged here , so it contains the different KBS